### DISCUSSION TITLE
Setting up on Openlitespeed

### DISCUSSION THREAD

**Discussion by mostfurious** [shlinkio/discussions#50] [Help wanted] [answered]
* Shlink Version: 2.10.1
* PHP Version: lsphp80
* How do you serve Shlink: Self-hosted Openlitespeed
* Database engine used: MariaDB

Hello,
If possible, i`d like to know how do you set it up with openlitespeed ? In virtualhost i pointed it to shlink/public folder, set permissions to "nobody" as are all other test folders like wordpress installations. But when i try to access the url i get "An unexpected error occurred"...
Tried running install file in bin folder but that did nothing ? Even tho it shows up green when using ls -l i tried to chmod +x and it still doesnt work.

**Comment by mostfurious** [comment, marked as answer]
Looks like releases are not tested, since i couldnt even start install script on Almalinux i moved to centos 7 vm and tried it there. When i run install i get `PHP Parse error:  syntax error, unexpected '|', expecting variable (T_VARIABLE) in /shlink/shlink2.10.1_php8.0_dist/vendor/shlinkio/shlink-common/functions/functions.php on line 49`

**Reply by acelaya** [reply]
It looks more like you are not using a supported PHP version.

Only PHP 8.0 and 8.1 are supported.
